### ISSUE TITLE
Access to prevProps from Presenter update

### ISSUE THREAD

**Issue by greypants** [vigetlabs/microcosm#175]
It appears that `this.props` is updated by the time `update` runs, meaning you can't do comparisons of previous and next props.

```js
Foo extends Presenter {
  update(repo, nextProps) {
    console.log('update', this.props === nextProps) // always true
    if(this.props.id !== nextProps.id) {
       // We'll never get in here because this.props === nextProps
    }
  }
}
```

**Comment by nhunzaker** [comment]
@greypants this should be fixed in `10.6.x`, could you verify?

**Comment by nhunzaker** [comment]
Okay this is definitely fixed. Closing!
